rudder-lang/src/parser.rs

```diff
@@ -599,9 +599,24 @@ fn pstruct(i: PInput) -> PResult<HashMap<String, PValue>> {
     )(i)
 }
 
+/// Alternative version of pstruct based on "." rather than braces.
+/// Used for the agent
+/// A struct is stored in a HashMap
+// fn pstruct_agent(i: PInput) -> PResult<HashMap<String, PValue>> {
```

Comment:
What is this ?

Reply:
I think you should remove this

Reply:
was removed from the next PR but I just removed it here too